lib/src/widgets/core/overlays/web_overlay.dart

```diff
@@ -150,8 +150,8 @@ class _WebOverlayBottomControlles extends StatelessWidget {
                         _WebSettingsDropdown(tag: tag),
                         MaterialIconButton(
                           toolTipMesg: _podCtr.isFullScreen
-                              ? '${_podCtr.podPlayerLabels.exitFullScreen}${kIsWeb ? ' (f)' : ''}'
```

Comment:
shortcut keys needed to be displayed by default in web.
if there is values in these variables (`_podCtr.podPlayerLabels.exitFullScreen`) then only we should override